src/components/App.tsx

```diff
@@ -13,31 +13,55 @@ import PathwaysList from './PathwaysList';
 import { PathwayProvider } from './PathwayProvider';
 import { Pathway } from 'pathways-model';
 import useGetPathwaysService from './PathwaysService/PathwaysService';
+import FHIR from 'fhirclient';
+import demoRecords from '../fixtures/MaureenMcodeDemoPatientRecords.json';
```

Comment:
Could these two demo files be combined into one, and then the Patient resource be parsed out only if necessary? One hope I had with this was that the patient could very easily be swapped out

Reply:
Removed demo patient and just implemented a search for the patient resource.